types/event.go

```diff
@@ -296,8 +296,10 @@ func IsLegacyAPIKey(apiKey string) bool {
 	return huskyotlp.IsClassicApiKey(apiKey)
 }
 
-func GenerateSpanID(traceID string) string {
-	ts := wyhash.Hash([]byte("next"), hashSeed)
-	h := wyhash.Hash([]byte(traceID), uint64(ts))
-	return fmt.Sprintf("%016x", h)
+func GenerateSpanID() string {
+	id, err := uuid.NewV7()
+	if err != nil {
+		return ""
```

Comment:
I don't know what would happen to cause this function to return an error, but maybe it's worth logging it out in the caller so that we don't mysteriously setting all span Id to `""`

Reply:
A logger isn't available here but I can fall back to returning a random value as a string.